### DISCUSSION TITLE
Add integration with Homebridge

### DISCUSSION THREAD

**Discussion by asyba** [netalertx/NetAlertX#531] [Ideas]
There ls aready integration with Home Assistant, can we do for Homebridge ?

**Comment by jokob-sk** [comment]
Hi there,
I am happy to approve a PR to create a new publisher for Homebridge. I unfortunately don't have HB and I had a quick look at the documentation and I couldn't find straightforward examples. If interested, check out the Plugin documentation for Pi.Alert:

https://github.com/jokob-sk/Pi.Alert/tree/main/front/plugins#readme

You can check existing plugins, such as the webhook publisher, and create a similar one for a different API suite.

Hope that's helpful,
J

**Comment by jokob-sk** [comment]
Hi @asyba ,

If you have some time, and know how to code, I've created a walkthrough of creating a custom plugin: [![Watch the video](https://raw.githubusercontent.com/jokob-sk/NetAlertX/main/docs/img/YouTube_thumbnail.png)](https://youtu.be/cdbxlwiWhv8)

![🎥Watch the video](https://youtu.be/cdbxlwiWhv8)
